inception-search-mtas/src/main/java/de/tudarmstadt/ukp/inception/search/index/mtas/MtasUimaParser.java

```diff
@@ -259,6 +295,36 @@ public String printConfig()
     {
         return null;
     }
+    
+    
+    /**
+     * Takes in IRI for identifier and returns teh label String
+     * Eg: 
+     * InputParameter :- http://www.w3.org/TR/2003/PR-owl-guide-20031209/wine#RoseDAnjou
+     * Returned :- "Concept"+MtasToken.DELIMITER+RoseDAnjou
+     * @param iri
+     * @return
+     */
+    public String getUILabel(String iri) {
+    	
+    	StringBuffer labelStr  = new StringBuffer("");
```

Comment:
Better use StringBuilder (http://courses.cs.tau.ac.il/software1/1516a/recitations/resources/String,StringBuffer,StringBuilder.pdf)

Reply:
You do not need to pass the empty string